interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/test/TestInfoProvider.scala

```diff
@@ -11,4 +12,6 @@ trait TestInfoProvider {
 
   def prepareTestData(preliminaryTestData: PreliminaryScenarioTestData, scenario: CanonicalProcess): Either[String, ScenarioTestData]
 
+  def getTestViewParameters(scenario: CanonicalProcess): Map[String, List[Parameter]]
```

Comment:
I know that it does not change anything, but I think it would be better to put this method after `getTestingCapabilities` as you did in the implementation :)

Reply:
If it makes code more readable then it is a good change :+1: